webant/templates/recents.html

```diff
@@ -0,0 +1,90 @@
+{% extends "bootstrap/base.html" %}
+{% import "bootstrap/fixes.html" as fixes %}
+
+{% block title %}
+Libreant | {%trans%}Recents{%endtrans%}
+{% endblock %}
+
+{% block styles %}
+ {{super()}}
+<link href="{{ url_for('static', filename='item_list.css') }}" rel="stylesheet">
+<style>
+header{
+    margin-top:10px;
+    margin-bottom: 20px;
+}
+header h1{
+    margin-top:0px;
+    margin-bottom: 0px;
+    font-size: 2.3em;
+    
+}
+</style>
+{% endblock %}
+
+{% block navbar %}
+{% import 'navbar.html' as navbar %}
+{{ navbar.navbar(search=True) }}
+{% endblock %}
+
+{% block content %}
+<div class="container">
+    <header>
+        <h1>{%trans%}Recently added items{%endtrans%}</h1>
+    </header>
+
+    {% if not items %}
+
+    {%trans%}Sorry, no items added since now{%endtrans%}
+
+    {% else %}
+
+    <div id="item-list">
+        {% for b in items %}
+        <div class="item-div row" onclick="window.open('{{url_for('view_book', bookid=b['_id'])}}')">
```

Comment:
>I asked you to provide the link from which you learned that ```onclick``` attribute is deprecated.

It is not deprecated per HTML standards, but it is considered evil when
you take the approach of unobtrusive javascript. Some resources:
* https://en.wikipedia.org/wiki/Unobtrusive_JavaScript
* https://en.wikipedia.org/wiki/Progressive_enhancement
Some of the main reason:
* using onclick violates the DRY principle: it writes the same code for
  each entry
* putting JS inside HTML violates the separation between functionality,
  presentation and content

For the same reason, I also don't like the usage of class names such as
`item-div`. I already know it's a div! Something like item-row-container
would be more semantic.

-- 
boyska